scheduled-jobs/scanning/art-notify/art-notify.py

```diff
@@ -76,14 +79,15 @@ def slack_link(job_name, job_url, job_id=None, text=None):
         return f"<{link}|{text}>"
 
     if failed_jobs:
-        failed_jobs.sort(key=lambda x: len(x[1]), reverse=True)
+        failed_jobs.sort(key=lambda x: x[2], reverse=True)
```

Comment:
```suggestion
        failed_text = ''
        failed_jobs.sort(key=lambda x: x[2], reverse=True)
```

Reply:
Ah I meant to add to text not failed_text